lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -13,18 +15,22 @@ def self.enabled
 
   def self.reset_priority_by_ordered_ids(ids)
     ids.each_with_index do |id, priority|
-      MiqAeDomain.where(:id => id).first.try(:update_attributes, :priority => priority + 1)
+      MiqAeDomain.find_by!(:id => id).update_attributes(:priority => priority + 1)
```

Comment:
@mkanoor This logic only works if the `ids` passed are all within the same tenant.  Should we check and raise an error if that is not true?